modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -32,860 +52,930 @@ These procedures are labeled with label:admin-only[].
+| *Mode* 3+| DBMS
+|===
 
-| xref:reference/procedures.adoc#procedure_dbms_setconfigvalue[`dbms.setConfigValue()`]
-| label:no[]
-| label:yes[]
-| label:admin-only[] label:not-aura[Not available on Aura]
+[role=label--enterprise-edition label--new-5.22 label--admin-only]
+[[procedure_dbms_cluster_switchdiscoveryserviceversion]]
+=== dbms.cluster.switchDiscoveryServiceVersion()
 
-| xref:reference/procedures.adoc#procedure_dbms_setDatabaseAllocator[`dbms.setDatabaseAllocator()`]
-| label:no[]
-| label:yes[]
-| label:admin-only[]
+.Details
+|===
+| *Syntax* 3+| `dbms.cluster.switchDiscoveryServiceVersion(mode)`
+| *Description* 3+| Allows you to select which discovery service should be started.
+Possible values are:
 
-| xref:reference/procedures.adoc#procedure_dbms_setDefaultAllocationNumbers[`dbms.setDefaultAllocationNumbers()`]
-| label:no[]
-| label:yes[]
-| label:admin-only[]
+* `V1_ONLY` -- it runs only discovery service v1.
+* `V1_OVER_V2` -- it runs both Discovery Service V1 and Discovery Service V2, where V1 is the main service and V2 runs in the background.
+* `V2_OVER_V1` -- it runs both Discovery Service V1 and Discovery Service V2, where V2 is the main service and V1 runs in the background.
+* `V2_ONLY` -- it runs only discovery service v2.
+.2+| *Input arguments* | *Name* | *Type* | *Description*
+| `mode` | `STRING` | mode :: STRING
+| *Mode* 3+| DBMS
+|===
 
-| xref:reference/procedures.adoc#procedure_dbms_setDefaultDatabase[`dbms.setDefaultDatabase()`]
-| label:no[]
-| label:yes[]
-| label:admin-only[]
+[role=label--enterprise-edition label--deprecated-5.23]
+[[procedure_dbms_cluster_uncordonServer]]
+=== dbms.cluster.uncordonServer()
 
+.Details
+|===
+| *Syntax* 3+| `dbms.cluster.uncordonServer(server)`
+| *Description* 3+| Remove the cordon on a server, returning it to 'enabled'.
+.2+| *Input arguments* | *Name* | *Type* | *Description*
+| `server` | `STRING` | The name or id of the server to be uncordoned.
+| *Mode* 3+| WRITE
+|===
 
-| xref:reference/procedures.adoc#procedure_dbms_showcurrentuser[`dbms.showCurrentUser()`]
-| label:yes[]
-| label:yes[]
-|
+Replaced by the xref:clustering/server-syntax.adoc#server-management-syntax[`ENABLE SERVER`] command.
 
-| xref:reference/procedures.adoc#procedure_dbms_showTopologyGraphConfig[`dbms.showTopologyGraphConfig()`]
-| label:no[]
-| label:yes[]
-| label:admin-only[]
+[NOTE]
+====
+Before Neo4j 5.23, the procedure `dbms.cluster.uncordonServer()` can be run only by users with `Admin` privileges.
+Since Neo4j 5.23, it can be run with the `SERVER MANAGEMENT` privilege.
+It will still run with the `Admin` privilege, but that should be considered deprecated.
+====
 
-// New in 5.19
-| xref:reference/procedures.adoc#procedure_genai_vector_listEncodingProviders[`genai.vector.listEncodingProviders()`]
-| label:yes[]
-| label:yes[]
-| label:new[Introduced in 5.19]
+[[procedure_dbms_routing_getroutingtable]]
+=== dbms.routing.getRoutingTable()
```

Comment:
This is not only cluster-related. And it looks like it is also available in CE.

Reply:
Ah, I now see that this section is not only for clusters. But it looks like this is the only procedure that is not cluster-related.